src/index.html

```diff
@@ -5,9 +5,322 @@
+    <div class="nav-mobile__top">
+      <img
+        src="./images/logo.png"
+        alt="Miami Logo"
+        class="nav-mobile__logo logo"
+      >
+      <button class="nav-mobile__times"></button>
+    </div>
+    <ul class="nav-mobile__list">
+      <li class="nav-mobile__item">
+        <a href="#header" class="nav-mobile__link">Home</a>
+      </li>
+      <li class="nav-mobile__item">
+        <a href="#info" class="nav-mobile__link">About Us</a>
+      </li>
+      <li class="nav-mobile__item">
+        <a href="#gallery" class="nav-mobile__link">Gallery</a>
+      </li>
+      <li class="nav-mobile__item">
+        <a href="#consultation" class="nav-mobile__link">Consultation</a>
+      </li>
+      <li class="nav-mobile__item">
+        <a href="#testimonials" class="nav-mobile__link">Testimonials</a>
+      </li>
+      <li class="nav-mobile__item">
+        <a href="#contact" class="nav-mobile__link">Contacts</a>
+      </li>
+    </ul>
+    <div class="nav-mobile__bottom">
+      <span class="nav-mobile__phone-number">+1 234 555-55-55</span>
+      <a href="tel:+1234555-55-55" class="nav-mobile__call">Call to order</a>
+    </div>
+  </nav>
+</header>
+<main>
+
+  <section class="section" id="info">
+    <div class="info">
+      <h1 class="info__heading">
+          Miami best<br>
+          Real Estate
+          <a class="link" href="#">
+            here
+          </a>
+      </h1>
+      <div class="info__text">
+        <h2 class="info__header">Over 50 years experience in love.</h2>
+        <p>
+          Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+        </p>
+        <p>
+          The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level doskanalno know this area better than other specialists.<br>
+          Working directly with developers - so have more useful information about the profitable deals.
+        </p>
+        <p>
+          <strong>We will never be late!<br>Guaranteed secure transaction - 100%</strong>
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="section" id="gallery">
+    <h2 class="heading section__heading">
+      Gallery
+      <span class="heading__crown"></span>
+    </h2>
+    <div class="gallery">
+        <div class="photos photos_wide_left">
+          <img
+            class="photos__img"
+            src="images/photos/1.jpg"
+            alt="Marina Palms / North Miami Beach, FL 33162"
+          >
+          <img
+            class="photos__img"
+            src="images/photos/2.jpg"
+            alt="Marina Palms / North Miami Beach, FL 33162"
+          >
+        </div>
+        <p>
+          <a class="link" href="https://bit.ly/3crwpcE">Marina Palms</a> / North Miami Beach, FL 33162
```

Comment:
не пиши все в одну строку
```suggestion
          <a class="link" href="https://bit.ly/3crwpcE">Marina Palms</a>
          <span> / North Miami Beach, FL 33162</span>
```